includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,290 @@
+			'♣' => [ 'title' => esc_html__( '♣', 'elementor' ) ],
+			'♦' => [ 'title' => esc_html__( '♦', 'elementor' ) ],
+		];
+
+		$this->start_controls_section(
+			'section_playing_card_title',
+			[
+				'label' => esc_html__( 'Playing Cards', 'elementor' ),
+			]
+		);
+
+		$repeater = new Repeater();
+
+		$repeater->add_control(
+			'playing_card_number',
+			[
+				'label' => esc_html__( 'Number', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'default' => 'K',
+				'placeholder' => esc_html__( 'Choose card number', 'elementor' ),
+				'dynamic' => [
+					'active' => true,
+				],
+				'options' => $card_numbers,
+			]
+		);
+
+		$repeater->add_control(
+			'playing_card_type',
+			[
+				'label' => esc_html__( 'Type', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'default' => '♥',
+				'placeholder' => esc_html__( 'Tab Title', 'elementor' ),
+				'dynamic' => [
+					'active' => true,
+				],
+				'options' => $card_types,
+			]
+		);
+
+		$this->add_control(
+			'cards',
+			[
+				'label' => esc_html__( 'Cards', 'elementor' ),
+				'type' => Controls_Manager::REPEATER,
+				'fields' => $repeater->get_controls(),
+				'default' => [
+					[
+						'playing_card_number' => esc_html__( 'A', 'elementor' ),
+						'playing_card_type' => esc_html__( '♠', 'elementor' ),
+					],
+				],
+				'title_field' => '{{{ playing_card_number }}} {{{ playing_card_type }}}',
+			]
+		);
+
+		$this->end_controls_section();
+
+		$this->start_controls_section(
+			'section_playing_cards_style',
+			[
+				'label' => esc_html__( 'Playing Cards', 'elementor' ),
+				'tab' => Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_control(
+			'playing_card_bg_color',
+			[
+				'label' => esc_html__( 'Background-Color', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'global' => [
+					'default' => '',
+				],
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-cards-wrapper .elementor-playing-card' => 'background-color: {{VALUE}};',
+				],
+			]
+		);
+
+		$this->add_control(
+			'playing_card_padding',
+			[
+				'label' => esc_html__( 'Padding', 'elementor' ),
+				'type' => Controls_Manager::DIMENSIONS,
+				'size_units' => [ 'px', '%', 'em', 'rem', 'vw', 'custom' ],
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-cards-wrapper .elementor-playing-card' => 'padding: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+				],
+			]
+		);
+
+		$this->add_control(
+			'playing_card_gap',
+			[
+				'label' => esc_html__( 'Gap', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', '%', 'em', 'rem', 'vw', 'custom' ],
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-cards-wrapper' => 'gap: {{SIZE}}{{UNIT}};',
+				],
+			]
+		);
+
+		$this->end_controls_section();
+
+	}
+	/**
+	 * Render HTML widget output on the frontend.
+	 *
+	 * Written in PHP and used to generate the final HTML.
+	 *
+	 * @since 1.0.0
```

Comment:
See above.

Reply:
Update these comments all over to the latest version. In the case of your widget it's since 3.16.0